mvc_controller.hpp

```diff
@@ -374,6 +374,9 @@ namespace model_view_controller{} // doxygen workaround.
 ///
 /// view_: Reference to View.
 ///
+/// itemboxes_cache_: Cached items of wxItemContainer instances 
```

Comment:
I'd like to make this comment slightly more precise:
```suggestion
/// itemboxes_cache_: Cached items of wxItemContainer instances, identified by their names,
```